### PR TITLE
TRM-27096: Improve advanced search examples

### PR DESCRIPTION
**Pull request by LeonardoGonzales**

### CODE REVIEW

File: uniprot-config/src/main/resources/search-fields-config/uniparc-search-fields.json [hunk 1]
```diff
@@ -159,7 +159,7 @@
     "dataType": "INTEGER",
     "sortFieldId": "length_sort",
     "description": "Search by length",
-    "example": "[100 TO 300]",
+    "example": "[50 TO 100]",
```
Comment (supun-ebi) [inline uniprot-config/src/main/resources/search-fields-config/uniparc-search-fields.json:162]: isnt it [1 TO 100] on spread sheet?

File: uniprot-config/src/main/resources/search-fields-config/uniprotkb-search-fields.json [hunk 2]
```diff
@@ -543,7 +543,7 @@
     "dataType": "STRING",
```
Comment (supun-ebi) [inline uniprot-config/src/main/resources/search-fields-config/uniprotkb-search-fields.json:543]: cannot see changes for pubmed_id fields

File: uniprot-config/src/main/resources/search-fields-config/uniref-search-fields.json [hunk 3]
```diff
@@ -60,7 +60,7 @@
     "fieldType": "GENERAL",
     "dataType": "ENUM",
     "description": "Search by identity",
-    "example": "sample identity",
+    "example": "1.0",
```
Comment (supun-ebi) [inline uniprot-config/src/main/resources/search-fields-config/uniref-search-fields.json:63]: 100% on spreadsheet?

Comment (LeonardoGonzales) [reply uniprot-config/src/main/resources/search-fields-config/uniref-search-fields.json:63]: The correct value is 1.0

File: uniprot-config/src/main/resources/search-fields-config/uniparc-search-fields.json (hunk 1 above)
Comment (LeonardoGonzales) [reply uniprot-config/src/main/resources/search-fields-config/uniparc-search-fields.json:162]: I tried to reduce a bit to a lighter query with less results

**Review by supun-ebi — APPROVED** [review]

**Pull request merged**